features/android/android_config.feature

```diff
@@ -7,3 +7,15 @@ Feature: Android Config
     When I run the game in the "AndroidPersistenceDirectory" state
     And I wait to receive an error
     And the exception "message" equals "Directory Found"
+
+  Scenario: Telemetry
+    When I run the game in the "AndroidBackgroundJVMSmokeTest" state
+    And I wait for 2 seconds
+    And I clear any error dialogue
```

Comment:
Is this really needed?

Reply:
The usage telemetry data will only be present in android and cocoa events, hence a seperate scenario needed to check it exists

Reply:
Here I meant the specific step `I clear any dialogue`

Reply:
Yes it might be, sometimes the OS shows the "this app keeps crashing" popup